streamly.cabal

```diff
@@ -696,6 +696,20 @@ test-suite internal-data-fold-test
     , QuickCheck        >= 2.14  && < 2.15
   default-language: Haskell2010
 
+test-suite internal-data-unfold-test
```

Comment:
we should have a consistent naming based on the module name like we have in benchmarks. This should be `Data.Unfold` or `Test.Data.Unfold`.